### PR TITLE
PEP8 fixes to notebook examples

### PR DESCRIPTION
**Pull request by mgserafi**
**Motivation and context:**
Addresses #495, fixing PEP8 spacing issues. Doesn't address reserved keywords, as discussed in the issue.

**How has this been tested?**
Made sure examples still run without errors.

**How long should this take to review?**
- Quick (less than 40 lines changed or changes are straightforward)

**Types of changes:**
- Non-code change (touches things like tests, documentation, build scripts)

**Checklist:**
- [X] I have read the **CONTRIBUTING.rst** document.
- [X] All new and existing tests passed.

### CODE REVIEW

**Comment by tbekolay** [comment]
Very cool thank you! I'm hoping you did this with a tool and not by hand yes? If there's a tool for it, we should make it part of our automated testing so that the notebooks stay well-styled :+1: If you point me to it I can a separate PR for that, or if you want to add it to this PR feel free -- its inclusion won't block this PR in either case.

**Comment by mgserafi** [comment]
Haha no no I didn't do it all by hand 😛 
I used a jupyter plugin called [code_prettify](https://github.com/jfbercher/code_prettify) which has an autopep8 thing for jupyter.

I'm not sure how to automate running it. I was running it within the notebook so I could see what changes it made. It would do a few things I didn't like such as remove the `;` and give errors when there's a `%` command.

**Comment by tbekolay** [comment]
Cool cool, I'll take a look and think about how to automate running it in a future PR. Thanks!

File: examples/advanced/inhibitory_gating.ipynb [hunk 1]
```diff
@@ -8,10 +8,11 @@
   {
    "cells": [
     {
-     "cell_type": "markdown",
+     "cell_type": "heading",
+     "level": 1,
      "metadata": {},
      "source": [
-      "# Nengo Example: Inhibitory Gating of Ensembles"
+      "Nengo Example: Inhibitory Gating of Ensembles"
```
Comment (jgosmann) [inline examples/advanced/inhibitory_gating.ipynb]: Is there a reason for changing the markdown headings to cells of the the type "heading"? Current Jupyter version do not use the "heading" cell type anymore. While these cells will automatically converted, I would prefer to keep the markdown cells.

Comment (mgserafi) [inline examples/advanced/inhibitory_gating.ipynb]: I don't think I explicitly changed this. It might've auto-changed when I opened it in jupyter v4 and converted back to v3?

Comment (jgosmann) [inline examples/advanced/inhibitory_gating.ipynb]: I added anothor commit to undo these changes.
